content/en/blog/2024/docs-localized.md

```diff
@@ -0,0 +1,42 @@
+---
+title: OpenTelemetry website goes multilingual!
+linkTitle: Multilingual website
+date: 2024-08-21
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco),
+issue: 4863
+sig: Comms
+cSpell:ignore: Comms
+---
+
+We are happy to announce that the OpenTelemetry documentation now supports
+multiple languages! Teams of contributors have already started to translate our
+resources into Portuguese (pt), Chinese (zh), and Japanese (ja).
```

Comment:
A few comments:

- Is it really worth adding the language ID?
- Let's maybe instead link each language to it's landing page.

Reply:
Agree on leaving out the language id.

Reply:
ah that's a great idea!